CMakeLists.txt

```diff
@@ -1,8 +1,12 @@
 cmake_minimum_required(VERSION 3.16)
-project(deepbacksub CXX)
+project(backscrub CXX)
```

Comment:
The name change should be its own separate commit.

Reply:
Good point, I'll put that elsewhere..